### ISSUE TITLE
Support for DPDK

### ISSUE THREAD

**Issue by teburd** [tailhook/rotor#22]
It would be awesome if there was a polling loop implementation for rotor's event monitoring that used DPDK and userland TCP.

http://www.seastar-project.org/ manages to show some amazing numbers (lies, and more lies I'm sure, or maybe not!) that make me wonder how nice it would be to have that same functionality but without the nightmarish build they have going on.

**Comment by tailhook** [comment]
Yes. I was trying to design `rotor-stream` with this use case in mind. But I'm pretty sure I will not have any time to work on this soon. And I also don't have any prior experience with this kind of thing.

Anyway, if there is anybody who wants to try, I will provide as much assistance as I can.
